src/DataBaseHelper.java

```diff
@@ -0,0 +1,32 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+/* Here I am storing all the user given data and results along with data and time of the operation into database*/
+class DataBaseHelper {
+    
+   protected DataBaseHelper(String word, String filepath, int totalNoOfWords, String errorMessage) throws SQLException {
+        storingDataToDataBase(word, filepath, totalNoOfWords, errorMessage);
+    }
+
+    private static void storingDataToDataBase(String word, String filepath, int totalNoOfWords, String errorMessage) throws SQLException {
+        Connection connectionToDataBase = null;
+        try {
+            DateTimeFormatter dateAndTimeFormat = DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss");
```

Comment:
add this "yyyy/MM/dd HH:mm:ss" String pattern into a date String variable

Reply:
Now I am storing all the string into respected variables